### PR TITLE
translated versinfo.cpp, toolbar8, allochan.cpp

### PR DESCRIPTION
**Pull request by 0xeb**
Please @janrysavy , let's get this moving fast.

### CODE REVIEW

File: src/versinfo.cpp [hunk 1]
```diff
@@ -186,7 +187,7 @@ CVersionInfo::LoadBlock(const BYTE*& ptr, CVersionBlock* parent)
     {
         int valSize = info->wValueLength;
 
-        if (block->Type == vbtVersionInfo) // udelame par kontrol konzistence dat
+        if (block->Type == vbtVersionInfo) // Let's perform some consistency checks on the data
```
Comment (janrysavy) [inline src/versinfo.cpp:190]: Let's -> let's

**Review by janrysavy — COMMENTED** [review]
There is some lower/upper case inconsistency, in some cases GPT-4 translation is better, some broken comment format. Overall, it looks very usable.

File: src/common/allochan.cpp [hunk 2]
```diff
@@ -28,15 +29,15 @@
...
 // that we need to call for initialization.
 
-#pragma warning(disable : 4075) // chceme definovat poradi inicializace modulu
+#pragma warning(disable : 4075) // we want to define the order of module initialization
 
 typedef void(__cdecl* _PVFV)(void);
 
 #pragma section(".i_alc$a", read)
-__declspec(allocate(".i_alc$a")) const _PVFV i_allochan = (_PVFV)1; // na zacatek sekce .i_alc si dame promennou i_allochan
+__declspec(allocate(".i_alc$a")) const _PVFV i_allochan = (_PVFV)1; // at the beginning of the section, we will use the variable i_allochan for .i_alc
```
[the remark this answers is not in the document]
Comment (0xeb) [reply src/common/allochan.cpp:37]: "we put" vs "we use" ?

File: src/common/allochan.cpp [hunk 3]
```diff
@@ -112,7 +113,7 @@ int C__AllocHandlerInit::AltapNewHandler(size_t size)
             if (res == 0)
             {
                 TRACE_ET(_T("AltapNewHandler: unable to open message-box!"));
-                Sleep(1000); // nechame masinu oddechnout a zkusime ukazat msgbox znovu
+                Sleep(1000); // Let the machine rest and try to show the msgbox again
```
[the remark this answers is not in the document]
Comment (0xeb) [reply src/common/allochan.cpp:116]: considering that the old comment is gone and new English one is in, is it not more correct to capitalize first character?

File: src/common/allochan.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (janrysavy) [reply src/common/allochan.cpp:37]: I mean GPT-3.5 "at the beginning of the section ... for .i_alc" vs GPT-4 "at the beginning of the .i_alc section ..."

**Comment by 0xeb** [comment]
Let me propose this workflow:

- I will give you commit access to my fork.
- Directly edit the changes and commit back to my fork/branch
- that will also reflect in the PR
- once we are good, we go back to your main branch and merge?

Like this, we both get commit credits (open source spirit), and also, it will be faster since you know the language.

We are dealing with 15k comments and it is more efficient if you directly start editing instead of commenting.

How does that sound?

File: src/toolbar8.cpp [hunk 4]
```diff
@@ -70,18 +70,17 @@ BOOL CPluginsBar::CreatePluginButtons()
     tii.Mask = TLBI_MASK_STYLE | TLBI_MASK_IMAGEINDEX | TLBI_MASK_ID;
     tii.Style = TLBI_STYLE_WHOLEDROPDOWN | TLBI_STYLE_DROPDOWN;
     tii.ImageIndex = i;
-    tii.ID = CM_PLUGINCMD_MIN + i; // do mainwnd3 prijde jako WM_USER_TBDROPDOWN
+    tii.ID = CM_PLUGINCMD_MIN + i; // will come to mainwnd3 as WM_USER_TBDROPDOWN
     InsertItem2(0xFFFFFFFF, TRUE, &tii);
-  }
-  */
+  }*/
```
[the remark this answers is not in the document]
Comment (0xeb) [reply src/toolbar8.cpp:75]: Ok, I will keep an eye as I do my review next time for new files.

File: src/common/allochan.cpp (hunk 3 above)
[the remark this answers is not in the document]
Comment (janrysavy) [reply src/common/allochan.cpp:116]: Are we able to ensure this consistently, across multi-line comments?

[the remark this answers is not in the document]
Comment (0xeb) [reply src/common/allochan.cpp:116]: Each "//" is translated on its own. I can attempt to write some logic to maintain capitalization or try to change my GPT prompt and see if it respects that.

[the remark this answers is not in the document]
Comment (0xeb) [reply src/common/allochan.cpp:116]: Let's see first how much this is wasting our time or how really important that is, before I re-do that efforts.

**Comment by janrysavy** [comment]
I should probably start by reviewing and closing the existing pull requests. This is a manual translation, using GPT-4 combined with DeepL. Unfortunately, I am not able to focus on this this week and next week, because I need to finish work for Fine.

**Comment by 0xeb** [comment]
1)

Manual GPT-4 is not going to be sustainable, not for anyone and not in any reasonable amount of time. I think it is fast to start off with the stuff I did, and just review / edit on the fly. It should take you (as a native speaker) no more than 10 minutes per file and we have around 170. So that's reasonable.

2)
What is DeepL?

3)
Please edit this PR at least per the workflow I suggested. They are 3 tiny files.

I need to start focusing on Unicode/UTF-16 support.

**Comment by 0xeb** [comment]
image
Note the first and last lines in this screenshot. I can quickly write a script to update the translation based on the original text's first word if upper case or not. I will update that through and generate new translation files for new files (not these 3 files).

**Pull request closed without merging**